pkg/minishift/addon/command/openshift_command.go

```diff
@@ -36,13 +36,13 @@ func NewOpenShiftCommand(command string, ignoreError bool) *OpenShiftCommand {
 }
 
 func (c *OpenShiftCommand) doExecute(ec *ExecutionContext, ignoreError bool) error {
-	// split off the actual 'oc' command. We are using our cached oc version to run oc commands
+	// split off the actual 'openshift' command. We are using origin container to run those commands
 	cmd := strings.Replace(c.rawCommand, "openshift ", "", 1)
 	cmd = ec.Interpolate(cmd)
 	fmt.Print(".")
 
 	commander := ec.GetDockerCommander()
-	_, err := commander.Exec("-t", constants.OpenshiftContainerName, constants.OpenshiftExec, ec.Interpolate(cmd))
+	_, err := commander.Exec("-t", constants.OpenshiftContainerName, constants.OpenshiftOcExec, ec.Interpolate(cmd))
```

Comment:
Why are we using `oc` here? Shouldn't we use `openshift` as it is for `openshift` command in add-on?

Reply:
No, all commands which are running inside the container should be part of OpenShift commander and we can separate oc commander (which interact with what we downloaded and cached) and OpenShift commander which interact with origin container.

Reply:
@praveenkumar I do not understand. As per documentation the openshift command should this `If the add-on command starts with openshift, you can run the openshift binary within the container that runs OpenShift. This means that any file parameters or other system-specific parameters must match the environment of the container instead of your host.` And I am not how `oc` can do this?